apps/webapp/app/services/externalApis/integrations/linear.ts

```diff
@@ -0,0 +1,128 @@
+import type { HelpSample, Integration } from "../types";
+
+const usageSample: HelpSample = {
+  title: "Using the client",
+  code: `
+import { Linear, events } from "@trigger.dev/linear";
+
+const linear = new Linear({
+  id: "__SLUG__",
+  token: process.env.LINEAR_TOKEN!,
```

Comment:
For the OAuth sample, we don't want this `token` line. 

I created a function in the Airtable one to deal with this: https://github.com/triggerdotdev/trigger.dev/blob/cd70a299707ccaa3657e1c271e2a593a72908fe8/apps/webapp/app/services/externalApis/integrations/airtable.ts#L3